app/controllers/api/v1/lmss_controller.rb

```diff
@@ -1,30 +1,68 @@
 module Api
   module V1
     class LmssController < BaseController
-      before_action :validate_name!, only: [:create]
-
-      def create
-        render :json => 'not yet implemented'.to_json, status: 501
-      end
+      before_action :validate_ids!, only: [:create]
 
       def index
-        render :json => 'not yet implemented'.to_json, status: 501
+        render json: 'not yet implemented', status: 501
```

Comment:
is this a legit use of `render :json`?  calling `to_json` on a plain string (which is what this call does) will not return a valid JSON object.  if you want a valid JSON object you need to be passing either a hash or something that converts straightforwardly to a set of key-value pairs.  if you really want to return just a string, you should use `render :text` instead.